.github/workflows/build.yml

```diff
@@ -0,0 +1,364 @@
+name: Build
+on:
+  pull_request: {}
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '* 0 * * *'
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    env:
+      CI: 'true'
+    steps:
+      - name: Set up Git
+        run: |
+          git config --global user.name "Tomster"
+          git config --global user.email "tomster@emberjs.com"
+      - name: Set up Volta
+        uses: rwjblue/setup-volta@v1
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Install dependencies
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y tree
+          yarn install
+          yarn upgrade ember-cli
+      - name: Lint (markdown source)
+        run: yarn lint:md:src
+      - name: Lint (typescript)
+        run: yarn lint:ts
+      - name: Compile (typescript)
+        run: yarn tsc
+      - name: Generate
+        run: yarn generate
+        env:
+          # This is needed for external PRs to build, since secure ENV don't apply to those
+          # Get your own token at https://account.mapbox.com/access-tokens
+          MAPBOX_ACCESS_TOKEN: 'pk.eyJ1IjoiZW1iZXJqcyIsImEiOiJjazBydnZjb2wwYXA5M2Rwc3IydGF2eXM0In0.EQiBFsRi9Jm70XFPiXnoHg'
+      - name: Lint (markdown output)
+        run: yarn lint:md:dist
+      - name: Prune artifacts
+        if: always()
+        working-directory: dist/code/super-rentals
+        run: git clean -dfX
+      - name: Upload artifacts (assets)
+        uses: actions/upload-artifact@master
+        if: always()
+        with:
+          name: assets
+          path: dist/assets
+      - name: Upload artifacts (markdown)
+        uses: actions/upload-artifact@master
+        if: always()
+        with:
+          name: markdown
+          path: dist/chapters
+      - name: Upload artifacts (code)
+        uses: actions/upload-artifact@master
+        if: always()
+        with:
+          name: code
+          path: dist/code/super-rentals
+
+  deploy-guides:
+    # This job deploys the markdown output the super-rentals-tutorial branch of
+    # the ember-learn/guides-source repository, along with the necessary assets
+    # (downloads and screenshots).
+    #
+    # For the markdown, we can simply copy over the output into the appropriate
+    # folder in guides-source. We start by emptying out the folder to account
+    # for any deleted or renamed chapters.
+    #
+    # For the downloads, it is also mostly a matter of copying them into the
+    # right location. As with the markdown content, we empty out the folder
+    # beforehand to account for deleted and renamed files.
+    #
+    # The JSON data used in the tutorial is stored as loose files here, to make
```

Comment:
> The JSON data used in the tutorial is stored as loose files here

Can we clarify what `here` is in this context? I _think_ it is referring to `guides-source`?